### PR TITLE
adding get_validator_jobs

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_db/processing_job.py [hunk 1]
```diff
@@ -931,6 +907,27 @@ def children(self):
             for jid in qdb.sql_connection.TRN.execute_fetchflatten():
                 yield ProcessingJob(jid)
 
+    @property
+    def get_validator_jobs(self):
```
Comment (ElDeveloper) [inline qiita_db/processing_job.py]: The pythonic thing would be to name this property `validator_jobs`.

File: qiita_db/test/test_processing_job.py [hunk 2]
```diff
@@ -492,7 +492,7 @@ def test_complete_success(self):
         job.complete(True, artifacts_data=artifacts_data)
         self._wait_for_job(job)
         # Retrieve the job that is performing the validation:
-        val_job = qdb.processing_job.ProcessingJob(job.step.rsplit(" ", 1)[-1])
+        val_job = list(job.get_validator_jobs)[-1]
```
Comment (ElDeveloper) [inline qiita_db/test/test_processing_job.py]: Why is this checking only the last job?

File: qiita_db/processing_job.py (hunk 1 above)
Comment (antgonza) [reply qiita_db/processing_job.py]: K, will change.

File: qiita_db/test/test_processing_job.py (hunk 2 above)
Comment (antgonza) [reply qiita_db/test/test_processing_job.py]: Not sure, that's the test it was before ...

Comment (ElDeveloper) [reply qiita_db/test/test_processing_job.py]: haha ok.

**Comment by ElDeveloper** [comment]
Thanks!

**Pull request merged**